test/integration/connection-monitoring-and-pooling/connection.test.js

```diff
@@ -195,7 +195,7 @@ describe('Connection', function () {
       }
     });
 
-    it('should correctly connect to server using domain socket', {
+    it.skip('should correctly connect to server using domain socket', {
       metadata: { requires: { topology: 'single', os: '!win32' } },
```

Comment:
Should be an auth disabled requirement instead right?

Reply:
yes good catch!